test/gerkin/tests/features/data_exist.feature

```diff
@@ -0,0 +1,27 @@
+Feature: Searches records based on input parameters and returns boolean answer.
+  Request endpoint: POST /data/{code}/{version}/exist
+
+  Background:
+    Given database with the valid schema has been set up
+
+  Scenario: Receive "true" when record exist
+    When I make a POST request with a valid payload and ID <id>
```

Comment:
When it comes to exists query I don't think ID is only relevant field. It should be more like: 
`When I want to check if records with specific parameters exists in system`